invenio_rdm_records/mappings/v6/records/record-v1.0.0.json

```diff
@@ -4,85 +4,289 @@
       "date_detection": false,
       "numeric_detection": false,
       "properties": {
-        "_access": {
+        "_bucket": {
+          "enabled": false
+        },
+        "_recid": {
+          "type": "keyword"
+        },
+        "_conceptrecid": {
+          "type": "keyword"
+        },
+        "_visibility": {
+          "type": "keyword"
+        },
+        "_visibility_files": {
+          "type": "keyword"
+        },
+        "_embargo_date": {
+          "type": "date"
+        },
+        "_access_condition": {
           "type": "object",
           "properties": {
-            "metadata_restricted": {
-              "type": "boolean"
+            "condition": {
+              "type": "keyword"
             },
-            "files_restricted": {
-              "type": "boolean"
+            "link_validity": {
+              "type": "keyword"
             }
           }
         },
-        "_bucket": {
-          "enabled": false
-        },
-        "access_right": {
+        "_contact": {
           "type": "keyword"
         },
-        "additional_descriptions": {
-          "type": "nested",
+        "_community": {
+          "type": "object",
           "properties": {
-            "description": {
+            "primary": {
               "type": "text"
             },
-            "description_type": {
+            "secondary": {
+              "type": "text"
+            }
+          }
+        },
+        "_owners": {
+          "type": "integer"
+        },          
+        "_created_by": {
+          "type": "integer"
+        },      
+        "_files": {
+          "type": "object",
+          "properties": {
+            "bucket": {
               "type": "keyword"
             },
-            "lang": {
+            "key": {
+              "type": "keyword",
+              "copy_to": "filename"
+            },
+            "version_id": {
+              "type": "keyword"
+            },
+            "size": {
+              "type": "long"
+            },
+            "checksum": {
               "type": "keyword"
+            },
+            "previewer": {
+              "type": "keyword"
+            },
+            "type": {
+              "type": "keyword",
+              "copy_to": "filetype"
             }
           }
         },
-        "additional_titles": {
-          "type": "nested",
+        "_default_preview": {
+          "enabled": false
+        },
+        "_internal_notes": {
+          "type": "object",
           "properties": {
-            "title": {
-              "type": "text",
-              "copy_to": "suggest_title"
+            "user": {
+              "type": "text"
```

Comment:
`keyword`, right? (to be able to associate a note with an internal user identifier)